actiontext/lib/action_text/attribute.rb

```diff
@@ -58,6 +75,55 @@ def #{name}=(body)
         scope :"with_rich_text_#{name}_and_embeds", -> { includes("rich_text_#{name}": { embeds_attachments: :blob }) }
       end
 
+      # Enable rich text support for a column of the model's table, instead of using an association.
+      # Example:
+      #
+      #   create_table :messages do |t|
+      #     t.string   :content
+      #   end
+      #
+      #   class Message < ActiveRecord::Base
+      #     rich_text_column :content
```

Comment:
I wouldn't document this in this way to prevent people from using this method directly and instead use the column option of the has_rich_text method